packages/typespec-ts/package.json

```diff
@@ -114,7 +114,7 @@
   },
   "peerDependencies": {
     "@azure-tools/typespec-azure-core": "^0.57.0",
-    "@azure-tools/typespec-client-generator-core": "^0.57.0",
+    "@azure-tools/typespec-client-generator-core": "^0.57.1",
```

Comment:
In the latest tcgc version the current RLC iterations would break among functions listOperationGroups and listOperationsInOperationGroup due to cache issues. So all RLC generation would fail.